server/graphql/v1/mutations/collectives.js

```diff
@@ -514,8 +514,9 @@ export async function archiveCollective(_, args, req) {
   }
 
   // TODO: cascade deactivation to EVENTs and PROJECTs?
+  await collective.changeHost(null);
 
-  return collective.update({ isActive: false, deactivatedAt: Date.now(), approvedAt: null, HostCollectiveId: null });
+  return collective.update({ isActive: false, deactivatedAt: Date.now(), approvedAt: null });
```

Comment:
:ice_cream: Good call to re-use `changeHost` here. We can probably simplify further given that `changeHost` resets `isActive`/`approvedAt`:

```suggestion
  return collective.update({ deactivatedAt: Date.now() });
```